jdbc/core/src/main/java/org/apache/shardingsphere/driver/jdbc/core/driver/ShardingSphereDriverURL.java

```diff
@@ -57,8 +60,16 @@ public ShardingSphereDriverURL(final String url) {
     @SneakyThrows(IOException.class)
     public byte[] toConfigurationBytes() {
         try (InputStream stream = inClasspath ? ShardingSphereDriverURL.class.getResourceAsStream("/" + file) : Files.newInputStream(new File(file).toPath())) {
-            byte[] result = new byte[null == stream ? 0 : stream.available()];
-            Objects.requireNonNull(stream, String.format("Can not find configuration file `%s`.", file)).read(result);
+            Objects.requireNonNull(stream, String.format("Can not find configuration file `%s`.", file)).read();
```

Comment:
Seems `...stream.read();` could be removed, data is already read by BufferedReader